content/modules/bpm/pages/bpm-concepts.adoc

```diff
@@ -0,0 +1,252 @@
+= Concepts
```

Comment:
Rename to "BPM Concepts" to not confuse with the framework's Concepts page

Reply:
Done